src/api/app/models/issue_tracker.rb

```diff
@@ -32,6 +33,14 @@ def write_to_backend
     self.issues_updated ||= Time.now
   end
 
+  def verify_name!(name)
```

Comment:
But....
If you say that the this validation really belongs to the **Issue** class... then move it!. If you don't want to instantiate an **Issue** only for validating the name, that's ok, then use a class method. The point is that the class responsible of knowing how a issue valid name is should be the **Issue** class, not the **IssueTracker** class.

Other thing is that is only validated against a Regexp if the IssueTracker instance is of kind "cve", that flow control... I prefer to have this outside the method. 

So,
```ruby
# then Issue model.is_valid_name? will return a boolean
  self.is_valid_name?(name)
    Regexp.new(/^\d\d\d\d-\d+$/).match(name)
  end

# Line 76 of patchinfo.rb can be
# And please change one char variable names to something easy to understand
raise IssueTracker::InvalidIssueName.new "The issue name is not supported: #{issue['id']}" if tracker.kind == 'cve' && !Issue.is_valid_name?(issue['id'])  
```